crates/store/re_chunk/src/chunk.rs

```diff
@@ -943,7 +934,11 @@ impl Chunk {
             entity_path,
             heap_size_bytes: Default::default(),
             is_sorted: true,
-            row_ids: Arrow2StructArray::new_empty(RowId::arrow2_datatype()),
+            row_ids: arrow::array::StructBuilder::from_fields(
```

Comment:
Here and elsewhere, it feels odd to me that we're going through a builder type just to build an empty thing.

Any particular reason we're not calling https://docs.rs/arrow/latest/arrow/array/fn.new_empty_array.html ?

Reply:
`new_empty_array` returns an `ArrayRef`, meaning we would need a downcast+unwrap here.
`StructBuilder` returns what we want.

Reply:
…but it may be worth it to get rid of `tuid_arrow_fields`